### ISSUE TITLE
Header and footer templates for NetCommunity pages

### ISSUE THREAD

**Issue by cchristopherhowardd** [aarome/aarome#303]
Questions are arising about how donation and ticket pages built and hosted in NetCommunity don't look like the new website and aren't mobile responsive. It may be time to start thinking about how to insert new code into these templates.

Examples of these pages include:

https://support.aarome.org/
https://support.aarome.org/2019-experience-packages
https://support.aarome.org/patrons-of-the-aar
https://support.aarome.org/annual-fund-spring-2020

My immediate thoughts are: (1) minimize the width of the header and footer; (2) minimize the content included in the header and footer; and (3) eliminate the right-side menu.

There are further complications. In addition to templates for these pages, there are a few forms with a process that take you through several pages before complete. Then there is the template for the confirmation form you land on after complete a transaction. There there is the acknowledgement email. But first things first.

**Comment by jasloe** [comment]
Can you grab the markup from the editor for one of the pages and put it in a [gist](https://gist.github.com/)? If each component is in a separate file, you can put each of them in a gist.

**Comment by cchristopherhowardd** [comment]
I'll look into the task this week. My initial understanding is that these templates are constructed from 10 to 20 different parts, some of which can be eliminated. For example, I see four separate sections that make up the header. We'd replace them with a single section that replicates the header of the new AAR website. We may want to reduce the words at the top (About, People, Events, etc.) in order to simulate a page that is web responsive.

<img width="1439" alt="Screen Shot 2020-05-26 at 10 55 07 AM" src="https://user-images.githubusercontent.com/38765846/82916105-9896d800-9f3f-11ea-8a3e-7b151d80e164.png">

**Comment by cchristopherhowardd** [comment]
The further I dig into this, the more I believe that we should build the donation and ticket pages from scratch. We shouldn't redesign the templates for the existing pages. We should design a template that simulates the new website and create NetCommunity pages only for the current campaigns. No retrofitting.

**Comment by jasloe** [comment]
Oh I totally agree. I just wanted to see what the NetCommunity markup looks like. When should we get started on this?

**Comment by cchristopherhowardd** [comment]
NetCommunity markup isn't easy to parse. Donate/ticket pages are built on templates, but some parts of these pages have HTML code and others don't (they interact with other components of NetCommunity. It's hard to explain and even harder to describe how they work. We may have to do a video conference call where I can share my screen.

This week is a short week and not good for getting started on the project for a few reasons--one of which is an email blast that I'll need your help with. This email is a LOT simpler than the one we did in April. It may just consist of an image file (NYC gala save the date) and a footer.

**Comment by cchristopherhowardd** [comment]
I am also making a list of NetCommunity webpages that are currently in use. I only want to focus on making sure the new template works on just those pages.

**Comment by jasloe** [comment]
Ok. Let me know. I'll be around all week. (Sending you an email now with updated travel dates.)

> This week is a short week and not good for getting started on the project for a few reasons--one of which is an email blast that I'll need your help with. This email is a LOT simpler than the one we did in April. It may just consist of an image file (NYC gala save the date) and a footer.

**Comment by cchristopherhowardd** [comment]
I pasted the CSS for NetCommunity pages to https://gist.github.com/cchristopherhowardd/75905087106c55694edcc7896ed0a76a

Can you access that?

**Comment by jasloe** [comment]
Yep. I don't know exactly what this would entail but is it possible to
create a new template that strips out the following markup:

headerUtility
headerLogo
contentSecondary
wrapFooter

With these components out of the way, my hope is that it would be
straightforward to adapt stylesheets that are close to the Open design.

I imagine each of these are stored as a separate component and assembled by
a master template. Or something like that?

**Comment by cchristopherhowardd** [comment]
> Yep. I don't know exactly what this would entail but is it possible to create a new template that strips out the following markup: headerUtility headerLogo contentSecondary wrapFooter

I imagine that it's possible to do this, but I don't quite have the programming expertise and vocabulary to express how. To prevent the NetCommunity site from breaking in ways that might take forever to fix, it may be a better idea to revise the existing template so that headerUtility and headerLogo are deleted, and something new is dropped in their place. 

I spent a few hours today under the hood and have a better understanding of how NetCommunity components fit together. Basically, most every (if not every) NetCommunity page is based on something called Primary Template - Two Column. If you make changes to this, you make changes site wide. I don't think doing this is a complicated project, but one must be precise. I've found that there's not a solid way to test work--if you save a change, it's live.

Last week I was required to change my NetCommunity password, which is now viHnJ6MtJ4hviqN. If you're so inclined, log in and poke around. Or we could do a Google Meet share screen and I can give you a tour of this museum piece.

It's my hope that with tweaks to Primary Template - Two Column and changes to the CSS, we can update NC to look something like the new AAR website, quickly and relatively easily. New header, new footer, correct typeface, matching colors.
